mxcube3/ui/serverIO.js

```diff
@@ -32,6 +37,10 @@ export default class ServerIO {
       }
     });
 
+    this.hwrSocket.on('beam_value_change', (data) => {
```

Comment:
It is not only position. It is also beam size and shape,  so we choose to use something more generic.